test/batchnorm.jl

```diff
@@ -0,0 +1,97 @@
+include("header.jl")
+
+srand(42)
+TOL=0.1
+
+# CPU 4d buffers
+ax4 = Array(5rand(4,5,5,3));
+aw4 = bnparam(5)
+# GPU 4d buffers
+kax4 = KnetArray(5rand(4,5,5,3));
+kaw4 = KnetArray(bnparam(5));
+
+# Array types
+at = typeof(ax4)
+kat = typeof(kax4)
+
+# utils
+stdev(x) = mean((x .- mean(x)) .* (x .- mean(x)))
```

Comment:
1. This seems like variance.
2. There should be a standard Julia function for this, var()?

Reply:
1.  Yes, stdev is not the correct name, so I'll change it. The stdev is only used for a sanity check  and the result should be 1, so test is still valid.
2.  There was no `var` function that takes `KnetArray` when the last time I merged with the upstream, so I performed this operation. I can make it more efficient when I increase the number of tests.